ledger/sandbox/src/main/scala/com/digitalasset/platform/sandbox/stores/ActiveContracts.scala

```diff
@@ -178,11 +193,19 @@ class ActiveContractsManager[ACS](
             }
         }
 
-    st.result
+    st.mapAcs(_ implicitlyDisclose globalImplicitDisclosure).result
   }
 
 }
 
+trait ActiveContractsSteps[+Self] { this: ActiveContractsSteps[Self] =>
```

Comment:
this name is pretty bad now. suggestion:

```
ActiveContractsStep => ActiveContracts
ActiveContracts => ActiveContractsInMemory
```

Reply:
e07c9390388e1db927546793f52dee963f422ffb